### PR TITLE
Windows hooks

### PR DESCRIPTION
**Pull request by lox**
This discusses how Windows `.bat` hooks are run, and also describes Git for Windows.

### CODE REVIEW

**Comment by lox** [comment]
Nice one @harrietgrace

**Comment by harrietgrace** [comment]
Cool! Sorry to remove your 'whilst', it's a british-ism and can be a little confusing for non-brit/non-aussies 😄

Unless you want to add more bits, I think this is ready to ship!

File: pages/agent/v3/hooks.md.erb [hunk 1]
```diff
@@ -29,6 +29,11 @@ Hooks are called in the following order:
 1. `pre-exit` (local) - runs before the job finishes, useful for performing cleanup tasks
 1. `pre-exit` (global) - runs before the job finishes, useful for performing cleanup tasks
 
+<div class="Docs__note">
+<p class="Docs__note__heading">Windows hooks</p>
+These hooks work on unix-based systems with bash in the path. If you're running windows agents, you'll need some windows-specific additions to get your hooks running. See <a href="hooks#hooks-on-windows">Hooks on Windows</a> for a detailed description.</p>
```
Comment (keithpitt) [inline pages/agent/v3/hooks.md.erb]: I think Windows here needs to be capitalised?

File: pages/agent/v3/hooks.md.erb [hunk 2]
```diff
@@ -82,3 +87,19 @@ HOOK_SCRIPT
 
 chmod +x .buildkite/hooks/pre-command
 ```
+
+## Hooks on Windows
+
+Agents runnings Windows require hook files to have either:
+
+* A `.bat` extension, and be written in [Windows Batch](https://en.wikipedia.org/wiki/Batch_file), or 
+* Access to <a href="windows#git-for-windows">Git for Windows, which includes bash</a>
```
Comment (keithpitt) [inline pages/agent/v3/hooks.md.erb]: I think the link here should be `/docs/agent/v3/windows`?

Comment (keithpitt) [reply pages/agent/v3/hooks.md.erb]: And ` <a href="windows#git-for-windows">Git for Windows, which includes bash</a> ` should probs just be ` <a href="windows#git-for-windows">Git for Windows</a> , which includes bash `

File: pages/agent/v3/windows.md.erb [hunk 3]
```diff
@@ -22,3 +22,7 @@ See the [Agent SSH Keys](/docs/agent/v3/ssh-keys) documentation for more details
 ## Configuration
 
 See the [configuration documentation](/docs/agent/v3/configuration) for an explanation of each configuration setting.
+
+## Git for Windows
+
+While the agent will work without git installed, you will require [Git for Windows](https://gitforwindows.org/) to interact with git or ssh. In addition to providing git and ssh, Git for Windows includes Bash for Windows and a variety of standard linux tools which are frequently required for [plugins](plugins).
```
Comment (keithpitt) [inline pages/agent/v3/windows.md.erb]: Link here needs to be `/docs/agent/v3/plugins` and `Git` and `SSH` should have the correct casing.

**Comment by keithpitt** [comment]
@lox just found a few niggly bits, looks good aside from that though!

**Comment by harrietgrace** [comment]
Addressed these extra bits in #203 😄

**Pull request merged**